### ISSUE TITLE
manifest for docker.io/jellyfin/jellyfin:latest not found

### ISSUE THREAD

**Issue by annabel82** [jellyfin/jellyfin#279]
Hi there,

When using docker pull jellyfin/jellyfin on Fedora Server 29 it says:

[root@appserver /]# docker pull jellyfin/jellyfin
Using default tag: latest
Trying to pull repository docker.io/jellyfin/jellyfin ...
Trying to pull repository registry.fedoraproject.org/jellyfin/jellyfin ...
Trying to pull repository quay.io/jellyfin/jellyfin ...
Trying to pull repository registry.access.redhat.com/jellyfin/jellyfin ...
Trying to pull repository registry.centos.org/jellyfin/jellyfin ...
Trying to pull repository docker.io/jellyfin/jellyfin ...
manifest for docker.io/jellyfin/jellyfin:latest not found

**Comment by BobOkisama** [comment]
Getting same error trying to update.

**Comment by RaymondSchnyder** [comment]
@annabel82 @BobOkisama 
https://old.reddit.com/r/jellyfin/comments/a9ftd1/jellyfin_statement_on_recent_code_changes/ecixuk7/

tldr: jellyfin has a weird license issue right now and all binary builds are offline till this is fully resolved.
That means: No Docker images atm.

**Comment by JustAMan** [comment]
As far as I can tell we have resolved license issues, but devs who are in charge of docker are resting now (@nvllsvm spent at least 4 work days' time since that license problem surfaced - that is, in last 2 days).

Plus there are some regressions compared to previous release, thus we are looking into them to be rectified before next release.

**Comment by BobOkisama** [comment]
Thanks for the quick update, appreciate you guys! Waiting with baited breath for each release for that nvidia compiled ffmpeg, as I just cannot get mine to compile ;P

**Comment by nvllsvm** [comment]
I've published the `dev` tag on Docker Hub; compiled from the latest master branch.

Broken things:
- new media scanning (previously scanned items are fine)
- playback progress

**Comment by annabel82** [comment]
Thanks ever so much <3
